Tasks/AzureFileCopy/task.json

```diff
@@ -37,26 +29,34 @@
       "helpMarkDown": "Absolute path of the source folder, or file on the local machine, or a UNC share. Expression should return a single folder or a file."
     },
     {
-      "name": "StorageAccount",
-      "type": "string",
-      "label": "Storage Account",
+      "name": "ConnectedServiceName",
```

Comment:
it will not we have not removed storage account from task.json I guess changes got jumbled here